collivery.php

```diff
@@ -3,19 +3,19 @@
 use MdsSupportingClasses\MdsColliveryService;
 
 define('_MDS_DIR_', __DIR__);
-define('MDS_VERSION', '3.7.4');
+define('MDS_VERSION', '3.8.0');
```

Comment:
Please bump up a major version number.
This is a major change.
4.0.0

Reply:
Updated it to 4.0.0.
Still need to update the tag from 3.8.0 to 4.0.0

Reply:
Added tag for 4.0.0